frontend/common/src/components/form/Select/Select.tsx

```diff
@@ -31,6 +49,40 @@ export interface SelectProps
   nullSelection?: string;
   /** Determine if it should track unsaved changes and render it */
   trackUnsaved?: boolean;
+  /** Determine if it should sort options in alphanumeric ascending order */
+  doNotSort?: boolean;
+}
+
+function sortOptions(options: OptGroupOrOption[]) {
+  const tempOptions = options.map((option: OptGroupOrOption) =>
+    Object.prototype.hasOwnProperty.call(option, "options")
```

Comment:
Was `"options" in options` not working here?  It does the same thing, but is a little more terse.

REF: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in

Reply:
Unfortunately, using the  `in` operator here results in this:

## Screenshot
![Screen Shot 2023-02-17 at 10 53 11](https://user-images.githubusercontent.com/3046459/219702035-7c28f9f5-ea73-4a8c-ac53-e11e9708812e.png)

instead of:

## Screenshot
![Screen Shot 2023-02-17 at 10 54 06](https://user-images.githubusercontent.com/3046459/219702030-c4b44101-7d9d-40c6-b8db-6b4ed29b399b.png)


> Unlike the [in](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in) operator, this method does not check for the specified property in the object's prototype chain.

REF: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty#description

Reply:
Dang, okay!